### ISSUE TITLE
[MU4 Issue] Double click to dock/undock panels?

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#14088]
**Describe the bug**

AFAICS, the only way to undock/dock the PLAY panel a(nd other panels?) is to drag it.

Is it possible to add a double-click function to allow panels to automatically dock or undock?

**Platform information**

OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-3334481768, revision: github-musescore-musescore-97fb85d

**Comment by Tantacrul** [comment]
Good idea. Moving to 4.x. Will reopen after 4.0 release. Thanks @rgreen5 !
